reference/7.2/Microsoft.PowerShell.Core/About/about_PowerShell_Config.md

```diff
@@ -39,358 +72,380 @@ installation.
-  user configurations, this is CurrentUser module directory.
+For **AllUsers** configurations, this sets the **LocalMachine** execution
+policy. For **CurrentUser** configurations, this sets the **CurrentUser**
+execution policy.
 
-This example shows a `PSModulePath` configuration for a Windows environment:
+The following example sets the execution policy of PowerShell to
+`RemoteSigned`.
 
 ```json
 {
-  "PSModulePath": "C:\\Program Files\\PowerShell\\6\\Modules"
+  "Microsoft.PowerShell:ExecutionPolicy": "RemoteSigned"
 }
 ```
 
-This example shows a `PSModulePath` configuration for a macOS or Linux
-environment:
+For more information, see [about_Execution_Policies][01].
 
-```json
-{
-  "PSModulePath": "/opt/powershell/6/Modules"
-}
-```
+### PowerShellPolicies
 
-This example shows embedding an environment variable in a `PSModulePath`
-configuration. Note that using the `HOME` environment variable and the `/`
-directory separator, this will work on Windows, macOS and Linux.
+Windows has several settings that can be managed by Group Policy. Usually,
+those setting are stored in the Windows registry. Those setting can also be
+defined in the `powershell.config.json` file.
 
-```json
-{
-  "PSModulePath": "%HOME%/Documents/PowerShell/Modules"
-}
-```
+The `PowerShellPolicies` is a JSON object that contains key-value pairs for the
+various policy settings. Those policy settings can also be listed at the root
+level of the JSON file, outside of the `PowerShellPolicies` object. This
+setting can contain the following subkeys:
 
-This example shows embedding an environment variable in a `PSModulePath`
-configuration that will only work on macOS and Linux:
+- `ConsoleSessionConfiguration`
+- `ModuleLogging`
+- `ProtectedEventLogging`
+- `ScriptBlockLogging`
+- `ScriptExecution`
+- `Transcription`
+- `UpdatableHelp`
+
+The `ScriptExecution` setting is used to set the PowerShell Execution Policy.
+This take precedence over the `ExecutionPolicy` setting described above.
+
+Example:
 
 ```json
 {
-  "PSModulePath": "%XDG_CONFIG_HOME%/powershell/Modules"
+    "PowerShellPolicies": {
+        "ScriptExecution": {
+            "ExecutionPolicy": "RemoteSigned"
+        }
+    }
 }
 ```
 
-> [!NOTE]
-> PowerShell variables cannot be embedded in `PSModulePath` configurations.
-> `PSModulePath` configurations on Linux and macOS are case-sensitive. A
-> `PSModulePath` configuration must use valid directory separators for the
-> platform. On macOS and Linux, this means `/`. On Windows, both `/` and `\`
-> will work.
+For descriptions of the other policy settings, see the descriptions in the
+[Common configuration settings][06] section.
 
-### ExperimentalFeatures
+On Windows, PowerShell looks for the settings in the registry. Any setting
+found in the registry have precedence. Next PowerShell reads the JSON
+configuration. Any settings found under `PowerShellPolicies` as long as they
+are not found in the registry. Finally, any remaining settings found at the
```

Comment:
This reads like a sentence that got chopped at some point, leaving the fragment "Any settings... in the registry."

Reply:
fixed